src/server/controller/RegistrationController.js

```diff
@@ -256,7 +266,8 @@ registration.createTeam = (req, res, next) => co(function*() {
     yield api.postModel(`event/${req.body.event}/team/`, req.user, teamData);
 
   if (req.file) {
-    yield api.uploadPicture(req.file, team.profilePic);
+    // TODO: This is not updated for cloudinary now because it is dead code that needs to be removed at some point
+    yield api.uploadFile(req.file, team.profilePic);
```

Comment:
When will it be removed?

Reply:
https://github.com/BreakOutEvent/breakout-frontend/issues/300